src/components/Graph/Graph.tsx

```diff
@@ -223,11 +241,12 @@ const GraphMemo: FC<GraphMemoProps> = memo(
                     nodeRefs.current[nodeName] = node;
                   }}
                   pathwayState={pathway.states[nodeName]}
-                  isCurrentNode={false}
+                  isCurrentNode={pathway.states[nodeName].label === currentNode?.label}
                   xCoordinate={nodeCoordinates[nodeName].x + parentWidth / 2}
                   yCoordinate={nodeCoordinates[nodeName].y}
                   expanded={Boolean(expanded[nodeName])}
                   onClick={onClickHandler}
+                  isChoiceOfCurrent={isBranchState(currentNode) && childOfCurrent}
```

Comment:
Another naming comment - to stay consistent with current naming conventions this would be if the node is a transition of the current node so I would rename to `isTransitionOfCurrentBranch`